### ISSUE TITLE
False negative SlevomatCodingStandard.ControlStructures.DisallowTrailingMultiLineTernaryOperator

### ISSUE THREAD

**Issue by rmikalkenas** [slevomat/coding-standard#1650]
Hello, looks like this sniff does not report error for such cases:

```php
function foo() {}

$a = $b === 'a' ? foo()
    : 'c';
```

```php
function foo() {}

$a = $b === 'a' ? foo(
) : 'c';
```

**Comment by kukulich** [comment]
It's intentional: https://github.com/slevomat/coding-standard/commit/f02675182fca6ba88e839a9433ebf4557e356015

The actual report matches the sniff name.

Your version does make sense but it will need another sniff.
